dev_apps/music-nga/views/home/index.html

```diff
@@ -5,10 +5,9 @@
   <title>Home</title>
   <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1">
 
-  <link rel="stylesheet" href="/components/gaia-fonts/style.css">
-  <link rel="stylesheet" href="/components/gaia-icons/gaia-icons-embedded.css">
-  <link rel="stylesheet" href="/components/gaia-theme/gaia-theme.css">
+  <link rel="stylesheet" href="app://theme.gaiamobile.org/shared/elements/gaia-theme/gaia-theme.css">
```

Comment:
Can you make this change to all the views before landing? I know this patch focuses on the "Home" view, but I'd like to try and keep things as consistent as possible as we go along :-)